go/libkb/globals.go

```diff
@@ -176,9 +175,8 @@ func (g *GlobalContext) Init() *GlobalContext {
 	g.createLoginState()
 	g.Resolver = NewResolver(g)
 	g.RateLimits = NewRateLimits(g)
-	g.CachedUserLoader = NewCachedUserLoader(g, CachedUserTimeout)
-	g.UserDeviceCache = NewUserDeviceCache(g)
-	g.makeFullSelfCacher()
+	g.upakLoader = NewUncachedUPAKLoader(g)
+	g.fullSelfer = NewNullFullSelfCache(g)
```

Comment:
Sounds good,  would match the line above it.

I didn't see the initialization in ConfigureCaches in the gh diff.